pkg/volume/csi/OWNERS

```diff
@@ -1,7 +1,11 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- jsafrane
-- saad-ali
-- vladimirvivien
-- davidz627
+- sig-storage-approvers
```

Comment:
Should we just delete this file and let it bubble up to pkg/volume/OWNERS?

Reply:
Sure.